integration-tests/general/threadsafe-enclave-same-signer/build.gradle

```diff
@@ -13,18 +13,21 @@ conclave {
     runtime = runtimeType
     useInternalDockerRegistry = true
 
+    //  Note that the content of thread_safe_enclave_private_key_same_signer.pem is the same as
+    //    thread_safe_enclave_private_key.pem in thread_safe_enclave module.
+    //  We avoid relative paths here as Gramine tests would fail otherwise
```

Comment:
Do we have a ticket for this? I presume this will affect users as well. What was the error?

Reply:
I think it is because when we build the enclave in Docker we are in a mounted directory and the "above" directories are not available. I will reproduce the issue.
But do we really want to fix it? I.e. can't we just tell the users to have the signing key inside the enclave dirs?